lib/spindrift/packager.py

```diff
@@ -789,12 +817,20 @@ def install_local_package(path, dependency):
 
             if os.path.isfile(source):
                 shutil.copyfile(source, destination)
-            else:
+            elif os.path.isdir(source):
                 shutil.copytree(
                     source,
                     destination,
                     ignore=shutil.ignore_patterns(*IGNORED),
                 )
+            else:
+                source = find_source_from_metadata(folder)
+                if source is not None:
```

Comment:
Probably need to throw an exception here if `source` is None so the user can see when the problem started.

Reply:
Decided to log a warning only.